src/ARAnchorManager.js

```diff
@@ -65,23 +67,22 @@ export default class ARAnchorManager extends EventDispatcher {
   }
 
   /**
-   * Delete anchor from the THREE.Object3D.
-   * @return {boolean} a flag that indicates if the anchor was correctly removed for
-   * the provided object3d.
+   * Delete anchor associated to the THREE.Object3D.
    */
   remove(object3d) {
     if (!(object3d instanceof Object3D)) {
       throw new Error('Invalid Object3D trying to remove anchor');
     };
     const anchor = this.object3DsToAnchors_.get(object3d);
     if (!anchor) { 
-      return false 
+      return this;
```

Comment:
Before it was returning false to indicate a failed remove, is that type of feedback something we care about? Throwing an error instead? No strong opinion.

Reply:
I added a more thorough description to a comment chain on Max's review.